lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9639,6 +9639,86 @@ namespace Js
         return superCtor;
     }
 
+    Var JavascriptOperators::OP_ImportCall(Var specifier, ScriptContext* scriptContext)
+    {
+        ModuleRecordBase *moduleRecordBase = nullptr;
+        SourceTextModuleRecord *moduleRecord = nullptr;
+
+        JavascriptFunction* pfuncCaller;
+        if (JavascriptStackWalker::GetCaller(&pfuncCaller, scriptContext) && pfuncCaller && pfuncCaller->IsScriptFunction())
+        {
+            FunctionBody* parentFuncBody = pfuncCaller->GetFunctionBody();
+            ModuleID id = parentFuncBody->GetModuleID();
```

Comment:
The ModuleID here has no relationship with Javascript modules.
This is in reference to different global scope (either global, or in a  HTML form).
The type should rename ModuleRootID so it is not being confused.